Tests/AppcuesKitTests/Experiences/TooltipWrapperViewFrameTests.swift

```diff
@@ -0,0 +1,399 @@
+//
+//  TooltipWrapperViewFrameTests.swift
+//  AppcuesKitTests
+//
+//  Created by Matt on 2023-03-03.
+//  Copyright © 2023 Appcues. All rights reserved.
+//
+
+import XCTest
+@testable import AppcuesKit
+
+class TooltipWrapperViewFrameTests: XCTestCase {
+
+    private static let iPhone14ProPortraitFrame = CGRect(x: 0, y: 0, width: 393, height: 852)
+    private static let iPhone14ProLandscapeFrame = CGRect(x: 0, y: 0, width: 852, height: 393)
+    private static let iPadPro12PortraitFrame = CGRect(x: 0, y: 0, width: 1024, height: 1366)
+
+    var view: TooltipWrapperView!
+
+    override func setUpWithError() throws {
+        view = TooltipWrapperView()
+    }
+
+    // MARK: - No Target Rectangle
+
+    func testNoTargetRectangleDefaultFrame() {
+        // Arrange
+        view.frame = Self.iPhone14ProPortraitFrame
+        view.preferredWidth = nil
+        view.preferredPosition = nil
+        view.pointerSize = nil
+        view.distanceFromTarget = 0
+        view.targetRectangle = nil
+        view.preferredContentSize = nil
+
+        // Act
+        view.positionContentView()
+
+        // Assert
+        XCTAssertEqual(
+            view.shadowWrappingView.frame,
+            CGRect(x: 8, y: 552, width: 377, height: 300),
+            "Inside the layout margins and at the bottom of the screen"
+        )
```

Comment:
Note to self and @andretortolano, I think we may still need handling around the default frame on Android in the no target rectangle case, to match.

Reply:
@mmaatttt refresh memory please, is this for the case where an `@appcues/tooltip` trait is used, but no `@appcues/target-rectangle` or `@appcues/target-element` exists at all (or any other rect metadata setting trait)?

In the case of a failed target-element trait, we do not fall back to a bottom anchored tooltip, correct? But rather a fatal experience error and end of the experience.

Reply:
Yes and yes! 👍